internal/services/containers/migration/kubernetes_cluster_node_pool.go

```diff
@@ -455,6 +455,16 @@ func (k KubernetesClusterNodePoolV0ToV1) Schema() map[string]*pluginsdk.Schema {
 						Type:     pluginsdk.TypeString,
 						Required: true,
 					},
+					"drain_timeout_in_minutes": {
+						Type:     pluginsdk.TypeInt,
+						Optional: true,
+						Default:  30,
+					},
+					"node_soak_duration_in_minutes": {
+						Type:     pluginsdk.TypeInt,
+						Optional: true,
+						Default:  0,
+					},
```

Comment:
Overlooked this in the last review, but state migrations refer to the schema at a specific point in time, so these need to be removed 
```suggestion
```